assets/questions/unknownAlphabet.js

```diff
@@ -0,0 +1,321 @@
+// Copyright 2017 The TIE Author, s. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Alien language alphabet.
+ */
+
+globalData.questions['unknownAlphabet'] = {  // eslint-disable-line dot-notation
+  title: 'Alien Language Alphabet',
```

Comment:
Why do you call this unknownAlphabet in the ID and Alien Language Alphabet elsewhere? Why not just Unknown Alphabet all over?